service/iot-service-samples/role-based-authorization-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/RoleBasedAuthenticationSample.java

```diff
@@ -74,6 +74,14 @@ public static void main(String[] args)
         runJobClientSample(iotHubHostName, credential);
 
         runDeviceMethodClientSample(iotHubHostName, credential, newDeviceId);
+
+        // For users of IoT Hubs deployed in the Fairfax private cloud, your service client's options must specify these
+        // authentication scopes. The default authentication scopes are valid for all other deployments in public or
+        // private clouds outside of Azure Stack deployments.
+        RegistryManagerOptions fairfaxHubOptions = RegistryManagerOptions
```

Comment:
Prefer not to call this `fairfaxHubOptions` because the object represents all options for registry manager. How about just `registryManagerOptions`?

Reply:
This is the name of the instance of RegistryManagerOptions, not the class itself. The point is to demonstrate how to build an options object that allows for you to connect to fairfax hubs

Reply:
I get that. I still think it is a misleading variable name. I'd rather you use a comment to specify why you are doing it, rather than use a specific variable name, when the options class is available for all kinds of options, not just auth scope.